packages/elements/src/slider/__test__/slider.event.test.js

```diff
@@ -66,8 +66,8 @@ describe('slider/Events', function () {
     await oneEvent(slider, 'mousedown');
     expect(isDragging(el)).to.be.true;
     expect(el.from).to.equal(calculateValue(el, 150).toFixed(0));
-
-    expect(el.to).to.equal(calculateValue(el, window.innerWidth - 100).toString());
+    // Android: Need to test by comparing different value
+    expect(isNear(el.to, calculateValue(el, window.innerWidth - 100).toString(), 1, true)).to.be.true;
```

Comment:
`isNear`'s 2nd argument accepts `number` not `string`.
There are a few more locations with the same issue.

Lastly, it also applies to v7, unfortunately...
```suggestion
    expect(isNear(el.to, calculateValue(el, window.innerWidth - 100), 1, true)).to.be.true;
```